(GH-2977) Set main plan ID correctly (#3014)
puppetlabs/bolt: documentation/experimental_features.md

`PlanFailure` and continues to execute the plan. If you've provided `_catch_errors => true`
to `wait()`, Bolt returns any errors raised and the plan continues to execute.

#### Viewing failing results in parallel blocks

When an action fails in a parallel block, such as a task returning an error, Bolt

> does not print the error result to the console. For example, the following plan

Reviewer: Hm, it prints the error as a warning...
```
lucy ~/githubs/bolt ((6f8e3c60...)) $ be bolt plan run bolt

Starting: plan bolt
Starting: command 'badcommand' on localhost
Finished: command 'badcommand' with 1 failure in 0.0 sec
Error in future '2': run_command 'badcommand' failed on 1 target [ID: errored_futures]
Finished: plan bolt in 1.07 sec
parallel block failed on 1 target
  (file: /home/lucy/githubs/bolt/plans/init.pp, line: 5, column: 14)
```

runs a non-existent command on targets in parallel:

```puppet
plan example (
  TargetSpec $targets
) {
  $_targets = get_targets($targets)

  $results = parallelize($_targets) |$target| {
    run_command('badcommand', $target)
  }

  return $results
}
```

> The error result from each target the command is run is not printed to the console,
> making it difficult to know why the command failed:

Reviewer 2: ```suggestion
Bolt does not print the error result from the targets the command is run on,
making it difficult to know why the command failed:
```

Reviewer 2: I think "to the console" is clear from the context here, so we can remove it to shorten this sentence a bit
